Update action.php

chtiland opened the pull request:
Problem with toolbar's icon when wiki is in a "subfolder" of a domain (i.e. : http://www.domain.com/wiki/).

haobug commented:
Thanks for your contribution

haobug commented:
@chtiland have you confirmed new. Change locally?

chtiland commented:
@haobug I forced reinstallation of interwiki plugin (as date in **plugin.info.txt** and dokuwiki page are still "2019-03-19"), cleared all caches (wiki and browser), works fine !

By the way, Dokuwiki's interwiki page shows that this plugin is not compatible with "Hogfather", but I use HogFather and interwiki plugin works without any problem.

Thanks !

haobug commented:
> @haobug I forced reinstallation of interwiki plugin (as date in **plugin.info.txt** and dokuwiki page are still "2019-03-19"), cleared all caches (wiki and browser), works fine !

i have updated date field within plugin.info.txt to 2021-02-16.
to force plugin installation you can use the mannual install method in extension manager.

haobug commented:
> By the way, Dokuwiki's interwiki page shows that this plugin is not compatible with "Hogfather", but I use HogFather and interwiki plugin works without any problem.

i can't login into the dokuwiki official wiki. additionally the support status looks based on statistics of user anonymous data.

chtiland commented:
> i have updated date field within plugin.info.txt to 2021-02-16

Thanks !
 
> i can't login into the dokuwiki official wiki. additionally the support status looks based on statistics of user anonymous data.

I've updated dokuwiki's wiki for _lastupdate_ date and _Hogfather_ compatibility for interwiki plugin. 😉

haobug commented:
> I've updated dokuwiki's wiki for _lastupdate_ date and _Hogfather_ compatibility for interwiki plugin.

thanks for that
